RELEASES.md

```diff
@@ -0,0 +1,78 @@
+# Regular Releases
+
+## Motivation
+Developers are making rapid progress on Substrate. They should not be throttled with the overhead of a regular release schedule now. Nevertheless, Substrate is already stable enough to build robust blockchain solutions. The userbase and devex team would benefit from regular releases.
+
+## Solution
+The solution is for the devex team to tell their own release story. Substrate package and its dependencies (git submodules) tell that story.
+
+# Goals
+As a builder on Substrate, I want to.
+* Reliably reproduce the results described in tutorials.
+* Clearly explain where I am stuck when I cannot reproduce results from tutorials.
+* Know that a tutorial will not switch target versions or otherwise mutate out from under me.
+* Easily update to newer releases of Substrate, Apps, etc when and only when I choose to do so.
+
+# Constituents
+Substrate Package is a meta-package that depends on other Substrate ecosystem tools through git submodules. Each release of Substrate Package represents a set of tools that have tested and known to work smoothly together.
+
+The submodules are:
+* Node template
+* Module tempalte
+* ink template
+* apps
+* ui tempalte
+
+# Release Plan
+
+Substrate Package uses a three point versioning system with **M**ajor, **m**inor, and **p**oint releases `M.m.p`. The criteria for making a new release are.
+
+## For **M**ajor Releases
+Major releases are made when and only when Substrate has a new major release upstream. These are like long term support releases and all non-depricated docs will be updated when new major versions are released.
+
+## For **m**inor Releases
+Minor versions is where the Devex team tells their story. As new features come into constituent projects, Substrate Package will occasionally release stable working combinations. The criteria for making a new minor release are:
+
+* At least one month has passed since the previous minor or major release
+* At least one submodule contains a feature required for an already-written tutorial
+* The release works with most (not necessarily all) existing docs
+* What else?
```

Comment:
maybe we add some notion of backwards compatibility like the `semver` **m**inor? so how we define that in the context of our work...I like how you defined it in **p**oint release but maybe **m**inor has a looser definition that allows for the addition of APIs and tutorials, but doesn't break any existing docs/examples/tutorials?

If **m**inor is not *backwards compatible* based on how I just defined it, we should make another category because this would imply a significant gap between **m**inor and **p**oint that is unaccounted for and could be useful to have when we add new content but don't have to change existing content.